extmod/modbluetooth.c

```diff
@@ -281,6 +287,8 @@ STATIC MP_DEFINE_CONST_FUN_OBJ_VAR_BETWEEN(bluetooth_ble_active_obj, 1, 2, bluet
 STATIC mp_obj_t bluetooth_ble_config(size_t n_args, const mp_obj_t *args, mp_map_t *kwargs) {
     mp_obj_bluetooth_ble_t *self = MP_OBJ_TO_PTR(args[0]);
 
+    bluetooth_handle_uninitialized();
```

Comment:
Some configuration values might need to be set before the device is enabled.  Eg `rxbuf` settings below.  And some ports can return the MAC address without being enabled, which can also be a useful feature.  I'm not sure what the best way is to deal with this.  Maybe for `mp_bluetooth_get_device_addr()` it could return a status code, if it failed then this function here raises an exception that the MAC could not be read?